gulpfile.js

```diff
@@ -5,6 +5,82 @@ const sass = require("gulp-sass");
 const postcss = require("gulp-postcss");
 const autoprefixer = require("autoprefixer");
 const sync = require("browser-sync").create();
+const del = require("del");
+const csso = require("csso");
+const rename = require("rename");
+const imagemin = require("gulp-imagemin");
+const webp = require("gulp-webp");
```

Comment:
Первая ошибка сборки:

![image](https://user-images.githubusercontent.com/17790771/90170578-f00f4c00-dda8-11ea-891a-0780c7cd7d81.png)

Константа `webp` объявлена дважды: здесь и в строке 66. Нужно что-то переименовать. Например константу с вызовом модуля **gulp-webp**:

```suggestion
const gulpWebp = require("gulp-webp");
```

Reply:
/